prereise/gather/demanddata/transportation_electrification/immediate.py

```diff
@@ -0,0 +1,194 @@
+# DELETED: comm_type, LOAD, vmtsc
+

```

Comment:
There are several comments like this that seem to pertain more to the code development process than the code itself, and can probably be removed.

Reply:
Yup this is absolutely correct. If you see comments like this please ignore them haha, they are more for me than anything.